modules/oc-adm-by-example-content.adoc

```diff
@@ -340,31 +340,72 @@ Create a new project
 
 
 
+== oc adm node-image create
+Create an ISO image for booting the nodes to be added to the target cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Create the ISO image and download it in the current folder
+  oc adm node-image create
+  
+  # Use a different assets folder
+  oc adm node-image create --dir=/tmp/assets
```

Comment:
same here : ```./oc adm node-image create --dir=/tmp/assets
error: stat nodes-config.yaml: no such file or directory```

Reply:
In this example the `nodes-config.yaml` it's expected to be in the `/tmp/assets` folder